docs/codeql/reusables/download-github-database.rst

```diff
@@ -0,0 +1,5 @@
+GitHub also stores CodeQL databases for over 200,000 repos on GitHub.com, which you can download using the API. The list of repos is constantly growing and evolving to make sure that it includes the most interesting codebases for security research.
+
```

Comment:
I think we're going to want to link out to the new download CodeQL database endpoint here, once the OpenAPI changes are published. Presumably that content will explain what the acceptable values for `<language>` are?

It may be worth giving a little more information about the endpoint here, since the GitHub CLI isn't the only way to download a database from GitHub.com. Then just use the GH CLI example below as an example of a way to use the endpoint.